client/src/components/AuditAdmin/Setup/Contests/ContestForm.tsx

```diff
@@ -481,6 +486,18 @@ const ContestForm: React.FC<IProps> = ({
                             </label>
                           </FormSection>
                         )}
+                        {isBatchComparison && isTargeted && (
```

Comment:
So pending ballots is defined on a per-contest basis right? I suppose that seems fine, is there a reason why we chose that over having it set at the audit level? I suppose it is better to have more flexibility but is there a known case where it could be different by contest?

Reply:
I assume it could be different if there are different ballot styles that include different contests. But this approach of defining it for each contest was specified by @MarkVVaudits, so he'd be the best person to answer.

Reply:
@carolinemodic, great question. Yes, what Jonah said -- applicable not only once ballot styles are explicitly supported, but in the current design where contests can span different sets of jurisdictions. Imagine auditing a U.S. House contest nested in a statewide audit: the number of pending ballots statewide might be completely immaterial to the House contest, depending on the underlying facts.